intro-to-R-tidyverse/README.md

```diff
@@ -0,0 +1,15 @@
+# Intro to R and Tidyverse Training Module
+
+This CCDL-designed module covers the introduction to R and the Tidyverse package. It is designed to be taught in 1.5 hours. 
```

Comment:
```suggestion
This CCDL-designed module introduces the R programming language and some [tidyverse](https://www.tidyverse.org/) packages. 
The tidyverse is a collection of packages designed for data science. 
It is designed to be taught in 1.5 hours. 
```
One sentence per line

Reply:
This is what I thought, but I was also a bit confused whether or not that should apply here given the format of the [RNA-seq README](https://github.com/AlexsLemonade/training-modules/tree/b92055b3292b38a1bf42e363d32e33b097f25541/RNA-seq).

Reply:
When the markdown renders like in what you've linked above, you won't see the new line as a break between the two sentences. If we look at the raw file, each sentence is on its own line: https://raw.githubusercontent.com/AlexsLemonade/training-modules/master/RNA-seq/README.md

Checkout the third paragraph of https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown for some more info.

Reply:
Ahh, understood. Thank you.